tests/integration/cloudformation/api/test_stacks.py

```diff
@@ -426,3 +426,29 @@ def test_prevent_resource_deletion(deploy_cfn_template, cfn_client, sns_client,
     cfn_client.delete_stack(StackName=stack.stack_name)
 
     sns_client.get_topic_attributes(TopicArn=stack.outputs["TopicArn"])
+
+
+def test_updating_an_updated_stack_sets_status(deploy_cfn_template, cfn_client):
```

Comment:
```suggestion
@pytest.mark.aws_validated
def test_updating_an_updated_stack_sets_status(deploy_cfn_template, cfn_client):
```

If this is validated to run against AWS, would be great if you could add the marker :+1: 
Just a small convention since this wasn't always as normal as it might seem now and there's still a lot of legacy that doesn't work against AWS

Reply:
Thanks - I've [converted` the test to a snapshot test](https://github.com/localstack/localstack/pull/7272/commits/d327703a3ec7e7a4a21a8b1fe3f534186888c2e5) which I think is preferable.